apps/civil/civil-sdt/demo-00.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: civil-sdt
+  namespace: civil
+spec:
+  values:
+    java:
+      image: hmctspublic.azurecr.io/civil/sdt:prod-61f79fc-20231002103124 #{"$imagepolicy": "flux-system:demo-civil-sdt"}
+      environment:
+        PURGE_SCHEDULE_CRON: 0 15 0 * * 3,4,5
```

Comment:
Purge job needs to run 7 days a week.  Suggest splitting it up as 1, 2 and 3 on one instance and 4, 5, 6 and 7 on the other.

Reply:
done